### ISSUE TITLE
Pacaur should warn if a package does not exist when installing multiple packages

### ISSUE THREAD

**Issue by aouelete** [rmarquis/pacaur#541]
For example:
`pacaur -S dropbox spotify firefox-arora` will install dropbox and spotify, but simply ignore the mistyped `firefox-arora`. Instead, pacaur should warn the user if it did not find a package.

**Comment by rmarquis** [comment]
It isn't ignored, but searched in the AUR instead, and correctly warns about the package not being found in AUR.

**Comment by aouelete** [comment]
Hm, this is the output I get (with the latest version of pacaur):
```
> pacaur -S firefox-arora linux-git
:: Package(s) firefox-arora linux-git not found in repositories, trying AUR...
:: resolving dependencies...
:: looking for inter-conflicts...

AUR Packages  (1) linux-git-latest
Repo Packages (1) xmlto-0.0.28-1

Repo Download Size:   0.02 MiB
Repo Installed Size:  0.12 MiB

:: Proceed with installation? [Y/n]
```

If you have one package that is mistyped and several others that do exist in the AUR, then there is no warning that the package that was mistyped is not being installed.

**Comment by rmarquis** [comment]
This is the expected output, and the one that I indeed have:

```
$ pacaur -S dropbox spotify firefox-arora
:: Package(s) dropbox spotify firefox-arora not found in repositories, trying AUR...
:: resolving dependencies...
:: no results found for firefox-arora
```

```
$ pacaur -S firefox-arora linux-git
:: Package(s) firefox-arora linux-git not found in repositories, trying AUR...
:: resolving dependencies...
:: no results found for firefox-arora
```

Please read the bug report link and file necessary information.

**Comment by aouelete** [comment]
Ok, it seems like this has to do with the `IgnorePkg` option in `/etc/pacman.conf`.

With `IgnorePkg = pacman-mirrorlist`, I get the output I gave earlier and this [`bash -x pacaur -S firefox-arora`](https://gist.githubusercontent.com/aouelete/7336cc08e4ea61b113be1c179cadcd14/raw/f74ebea24e9f0a3741fbb232533bc014af3139a6/pacaur-with-ignore.txt).

Without that option, I get the correct output and this [`bash -x pacaur -S firefox-arora`](https://gist.githubusercontent.com/aouelete/7336cc08e4ea61b113be1c179cadcd14/raw/f74ebea24e9f0a3741fbb232533bc014af3139a6/pacaur-without-ignore.txt).

**Comment by rmarquis** [comment]
I see, will have a closer look asap.

**Comment by rmarquis** [comment]
Fixed in f5652957e66c493f388507386e7d69bce5c23a26.

**Comment by aouelete** [comment]
Thanks!
